backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -650,21 +650,23 @@ export class ExperimentAssignmentService {
           );
         })
       );
-
-      const allSites = [],
-        allTargets = [];
-      filteredExperiments.forEach((exp) => {
-        exp.partitions.forEach((partition) => {
-          allSites.push(partition.site);
-          allTargets.push(partition.target);
+      let monitoredLogCounts: any[];
```

Comment:
it would be safer to initialize this as an empty array, as later we call `.find` on it. It probably is not going to be undefined there, but why risk it. no point in giving type of it's `any`.

`let monitoredLogCounts = [];`